enterprise/server/remote_execution/executor/executor.go

```diff
@@ -158,7 +158,16 @@ func isClientBazel(task *repb.ExecutionTask) bool {
 	return !platform.IsCICommand(task.GetCommand(), platform.GetProto(task.GetAction(), task.GetCommand()))
 }
 
+func retriesEnabled(task *repb.ExecutionTask) bool {
+	v := platform.FindValue(task.GetPlatformOverrides(), platform.ShouldRetryPropertyName)
+	return v == "true" || v == ""
+}
+
 func shouldRetry(task *repb.ExecutionTask, taskError error) bool {
+	if !retriesEnabled(task) {
+		return false
+	}
```

Comment:
I think the scheduler server also has some retry logic, e.g. if the executor crashes before we hit this codepath, the scheduler still probably needs some way for the task to be attempted again. Have you looked at that / is there anything we need to change there?

Reply:
Good call - added some logic to address that